src/uct/ib/dc/dc_mlx5.h

```diff
@@ -240,6 +241,23 @@ typedef struct uct_dc_mlx5_ep_fc_entry {
 
 KHASH_MAP_INIT_INT64(uct_dc_mlx5_fc_hash, uct_dc_mlx5_ep_fc_entry_t);
 
+enum uct_dc_mlx5_dci_config_flags {
+    UCT_DC_MLX5_DCI_CONFIG_KEEPALIVE           = UCS_BIT(0),
+    UCT_DC_MLX5_DCI_CONFIG_MAX_RD_ATOMIC_IS_64 = UCS_BIT(1)
+};
+
+typedef union uct_dc_mlx5_dci_config {
+    struct {
+        uint8_t path_index;
+        uint8_t flags;
+        uint8_t padding[6];
+    } key;
+    uint64_t u64;
```

Comment:
IMO defining this as union is redundant since it is slow path code. it's ok even if it will not fit into 64 bit.
we can define helper function to calc 64-bit (or 32-bit) hash key from it instead